demos/TicTacToe/game_resources/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@
 
```

Comment:
you should also append the google copyright to the top, which needs to be an Apache license copyright.  See below.